src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './types';
+export * from './schema-builder';
```

Comment:
do you wanna expose all funcs from this file?
like `unwrapSchema`?
I see it's exported..

Reply:
Yeah, forgot about it, thanks.